afs/templates/views/components/reports/observation.htm

```diff
@@ -76,7 +73,52 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Obs
                     }
                 }"></div>
             </div>
-            <!-- Substance Tab -->
+            <!-- Related Objects Tab -->
+            <div class="afs-report-page" data-bind="if: activeSection() === 'objects'">
+                <div class="afs-report-section">
+                    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.relatedObjects)}, css: {'fa-angle-double-right': visible.relatedObjects(), 'fa-angle-double-up': !visible.relatedObjects()}" class="fa toggle"></i> {% trans "Related Objects" %}</h2>
+                      
+                    <!-- Collapsible content -->
+                    <div data-bind="visible: visible.relatedObjects" class="afs-report-collapsible-container pad-lft">
+
+                        <!-- ko ifnot: collectionOfRelatedObjects().length -->
+                        <div class="afs-nodata-note">No objects related to this observation</div>
+                        <!-- /ko -->
+
+                        <!-- ko if: collectionOfRelatedObjects().length -->
+                        <div class="afs-report-subsection" >
+                            <div>
+                                <div class="afs-table pad-btm">
+                                    <table class="table table-striped" cellspacing="0" width="100%">
+                                        <thead>
+                                            <tr class="afs-table-header">
+                                                <th>{% trans "Object" %}</th>
+                                                <th class="min-tabletl">{% trans "Description" %}</th>
+                                                <th class="afs-table-control all"></th>
+                                            </tr>
+                                        </thead>
+                                        <tbody data-bind="dataTablesForEach: { data: collectionOfRelatedObjects, dataTableOptions: relatedObjectsTableConfig }">
```

Comment:
This table does not seem to follow the typical patterns... as in 
https://github.com/archesproject/arches-for-science-prj/blob/dev/6.2.x/afs/templates/views/components/reports/scenes/documentation.htm#L56-L71

Reply:
Hi @njkim! I'm following the pattern seen in [project.htm](https://github.com/archesproject/arches-for-science-prj/blob/dev/6.2.x/afs/templates/views/components/reports/project.htm#L99-L111). Granted, I wrote that. I consulted with @aarongundel when diverging from that pattern.